inc/class-wp-cli-command.php

```diff
@@ -99,6 +99,39 @@ public function save( $args, $args_assoc ) {
 			if ( ! empty( $args_assoc['verbose'] ) ) {
 				WP_CLI::line( 'Saving ' . $url );
 			}
+
+			$options = [
+				'user_id' => get_current_user_id(),
+				'context' => '',
+				'action'  => '',
+			];
+
+			$post_id = url_to_postid( $url );
+			if ( ! empty( $post_id ) ) {
+				$post = get_post( $post_id );
+				if ( $post->post_type === 'page' ) {
+					$options['context'] = 'page';
+					$options['action']  = 'wp_cli_netstorage_publish';
+				}
+			} else {
+				$parsed_url = wp_parse_url( $url );
+				if ( ! empty( $parsed_url ) ) {
+					$args = array(
+						'meta_key'   => 'netstorage_path',
+						'meta_value' => substr( $parsed_url['path'], 1 ),
+						'post_type'  => 'netstorage-file',
+					);
+
+					$query = new \WP_Query( $args );
+					if ( ! empty( $query->post ) ) { // Get the first post assuming no posts will have the same path.
```

Comment:
Not sure about the comment here, since there does not seem to be any data used from the query.

Reply:
Added an additional check to make sure the `$post` matches the current `$post_id`